### DISCUSSION TITLE
[Feature] Focus "Filter Tags"

### DISCUSSION THREAD

**Discussion by dschaehi** [windingwind/zotero-actions-tags#433] [Ideas]
If I want to quickly filter the items by tags, I have to move my mouse to the tags pane and click on "Filter Tags" entry. This somehow interrupts my workflow.
It would be great if someone can write a script for focusing on "Filter Tags".

On a different note, for adding tags, there is this amazing script: https://github.com/windingwind/zotero-actions-tags/discussions/390#discussion-7226270

**Comment by mjthoraval** [comment]
Please test: https://github.com/windingwind/zotero-actions-tags/discussions/575
